ColabTool/Shared/PostFooter/style.css

```diff
@@ -9,16 +9,16 @@
 
 .post-action-details {
   flex-grow: 1;
-}
-
-.post-controls {
   display: flex;
+  align-items: center;
 }
 
-.post-icon {
+svg {
```

Comment:
It seems like this rule could be confusing since it could change any svg element on the page. Would be better to use a scoped class here